playground/backend/internal/environment/application.go

```diff
@@ -69,8 +74,13 @@ func NewCacheEnvs(cacheType, cacheAddress string, cacheExpirationTime time.Durat
 
 //ApplicationEnvs contains all environment variables that needed to run backend processes
 type ApplicationEnvs struct {
-	workingDir             string
-	cacheEnvs              *CacheEnvs
+	// workingDir is a root working directory of application
```

Comment:
```suggestion
	// workingDir is a root working directory of application..
	// This directory is different from the `pwd` of the application. It is a working directory passed as
	// a parameter where the application stores code, executables, etc.
```

Let me know if this is correct, and if it is, feel free to apply it? : )

Reply:
Changed.